patches/components-sync-driver-sync_auth_manager.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/sync/driver/sync_auth_manager.h b/components/sync/driver/sync_auth_manager.h
+index 72a72bee65056fae399b2bd5ca6dce065ddd519f..1ec376760d6fedcdb46b38a7915bbda76fcbade4 100644
+--- a/components/sync/driver/sync_auth_manager.h
++++ b/components/sync/driver/sync_auth_manager.h
+@@ -122,6 +122,7 @@ class SyncAuthManager : public signin::IdentityManager::Observer {
+   void ResetRequestAccessTokenBackoffForTest();
+ 
+  private:
++  BRAVE_SYNC_AUTH_MANAGER_H_
```

Comment:
I believe the technique in https://github.com/brave/brave-browser/wiki/Patching-Chromium#making-methods-virtual can also be used here

Reply:
fixed https://github.com/brave/brave-core/pull/5294/commits/b2fdf8c10dd8d42e0f3add64c775cfd12b8abd57